unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -728,4 +813,51 @@ private bool ShouldShowReplaceOtherWearablesToast(WearableItem wearable)
         }
         return true;
     }
+
+    private void InitializeEmotesSection(Transform currentViewTransform, Transform previousViewTransform)
+    {
+        if (currentViewTransform == null)
+            return;
+
+        emotesSectionTransform = currentViewTransform;
+        emotesSectionTransform.SetParent(view.emotesSection.transform, false);
+
+        view.SetSectionActive(AvatarEditorHUDView.EMOTES_SECTION_INDEX, true);
+    }
+
+    private void IsEmotesCustomizationSelectedChanged(bool current, bool previous)
+    {
+        if (!current)
+            return;
+
+        view.sectionSelector.GetSection(AvatarEditorHUDView.EMOTES_SECTION_INDEX).SelectToggle();
+    }
+
+    private void OnNewEmoteAdded(string emoteId)
+    {
+        if (!isAvatarPreviewReady)
+            return;
+
+        UpdateAvatarPreview();
+    }
+
+    private void OnPreviewEmote(string currentEmoteId, string previousEmoteId) { view.PlayPreviewEmote(currentEmoteId); }
+
+    private void OnEmoteEquipped(string currentEmoteId, string previousEmoteId)
+    {
+        catalog.TryGetValue(currentEmoteId, out WearableItem equippedEmote);
+
+        if (equippedEmote != null)
+            EquipEmote(equippedEmote);
+    }
+
+    private void OnEmoteUnequipped(string currentEmoteId, string previousEmoteId)
+    {
+        catalog.TryGetValue(currentEmoteId, out WearableItem unequippedEmote);
+
+        if (unequippedEmote != null)
+            UnequipEmote(unequippedEmote);
+    }
+
+    private void OnRedirectToEmoteSelling(string currentEmoteId, string previousEmoteId) { SellCollectible(currentEmoteId); }
```

Comment:
whenever i clicked sell on any of the emotes, it redirected to any specific nft, just: https://market.decentraland.org/account?section=collections
i would guess it is because the builtin emotes are not nfts therefore you cannot sell them?